libr/anal/sign.c

```diff
@@ -535,7 +585,9 @@ static bool addItem(RAnal *a, RSignItem *it) {
 		mergeItem (curit, it);
 		serialize (a, curit, key, val);
 	}
-	sdb_set (a->sdb_zigns, key, val, 0);
+	char *query = r_str_newf ("%s=%s", key, val);
```

Comment:
wat. why do a query, when you can just use the api?

Reply:
At that point I thought I had a bug in the code that was caused by the API when setting values containing commas.

It ended up being something unrelated and I forgot to undo this change.